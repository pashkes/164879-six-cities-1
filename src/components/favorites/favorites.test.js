import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Favorites from "./favorites.jsx";

configure({adapter: new Adapter()});

it(`favorites is renders correctly`, () => {
  const favorites = shallow(<Favorites history={{}} changeCity={jest.fn()} cities={[`Amsterdam`]} favorites={{'Amsterdam': []}} />);
  expect(favorites).toMatchSnapshot();
});
