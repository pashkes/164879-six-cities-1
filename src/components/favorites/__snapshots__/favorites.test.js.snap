// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`favorites is renders correctly 1`] = `
<ul
  aria-live="polite"
  className="favorites__list"
  role="status"
>
  <li
    className="favorites__locations-items"
    key="Amsterdam"
  >
    <div
      className="favorites__locations locations locations--current"
    >
      <div
        className="locations__item"
      >
        <a
          className="locations__item-link"
          href="#"
        >
          <span>
            Amsterdam
          </span>
        </a>
      </div>
    </div>
    <div
      aria-live="polite"
      className="favorites__places"
      role="status"
    />
  </li>
</ul>
`;
